segmentation/include/pcl/segmentation/impl/organized_connected_component_segmentation.hpp

```diff
@@ -205,8 +205,8 @@ pcl::OrganizedConnectedComponentSegmentation<PointT, PointLT>::segment (pcl::Poi
   }
   
   std::vector<unsigned> map (clust_id);
```

Comment:
Seems like `map` needs to be a `std::vector<size_t>` now.

Reply:
There are some issues like this now, but these changes are to big for a single PR. (if we e.g. change `map` here, we need to adjust `findRoot`, too). Maybe we should collect a list of Todos here (or in (a) issue(s)) to fix them later,